src/resources/form.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"
+      xmlns="http://www.w3.org/1999/html">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Rejestracja</title>
+</head>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous">
+
+<form style="font-family: 'Arial'; margin-left: 300px; margin-right: 300px" method="post" action="URL">
+    <p style="font-size: x-large"><b>Rejestracja</b></p>
+
+    <div class="form-group">
+        <label for="firstName"> Imię</label>
+        <input id="firstName" class="form-control" required="required" type="text" minlength="5"
+               maxlength="10" name="firstName"
+               placeholder="Wpisz imię"><br>
```

Comment:
Tak dodam tylko że wg mnie te tagi `br` powinny być zamykane od razu czyli `<br/>`

Reply:
Wiem że przeglądarka to wybacza, ale mimo wszyzstko.